--- tst-timezone.c	2012-01-20 15:07:25.254388843 +1000
+++ tst-timezone.c	2012-01-20 15:07:51.231556024 +1000
@@ -39,7 +39,7 @@
   { "Europe/Berlin", 1, -3600, { "CET", "CEST" }},
   { "Europe/London", 1, 0, { "GMT", "BST" }},
   { "Universal", 0, 0, {"UTC", "UTC" }},
-  { "Australia/Melbourne", 1, -36000, { "EST", "EST" }},
+  { "Australia/Melbourne", 1, -36000, { "AEST", "AEDT" }},
   { "America/Sao_Paulo", 1, 10800, {"BRT", "BRST" }},
   { "America/Chicago", 1, 21600, {"CST", "CDT" }},
   { "America/Indiana/Indianapolis", 1, 18000, {"EST", "EDT" }},
