--- tzdata.spec	2013-11-12 09:37:27.086625633 +1000
+++ tzdata.spec_patched	2013-11-12 09:52:53.689293626 +1000
@@ -10,6 +10,7 @@
 Source0: ftp://ftp.iana.org/tz/releases/tzdata%{tzdata_version}.tar.gz
 Source1: ftp://ftp.iana.org/tz/releases/tzcode%{tzcode_version}.tar.gz
 Patch1: tzdata-2011l-kemerovo.patch
+Patch2: australia-names.diff
 
 BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 BuildRequires: gawk, glibc, perl
@@ -20,7 +21,7 @@
 
 %description
 This package contains data files with rules for various timezones around
-the world.
+the world. It was patched to use AEST and AEDT instead of EST/EDT.
 
 %package java
 Summary: Timezone data for Java
@@ -35,6 +36,7 @@
 %setup -q -c -a 1
 
 %patch1 -p1
+%patch2 -p1
 
 mkdir javazic
 tar zxf %{SOURCE3} -C javazic
